denlillemand/week4/ass5/app/actioncreators/listactions.js

```diff
@@ -0,0 +1,70 @@
+import Dispatcher from '../dispatcher.js';
+import { CREATE_LIST, DELETE_MESSAGE, CREATE_MESSAGE, SWAP_MESSAGE, ARCHIVE_MESSAGE } from '../constants/listconstants.js';
+
+export function createList(listName) {
```

Comment:
I think you should create a class contain the functions that you need to export and then export that class. IMO, it would also make the consuming code more readable because you then get the Class.Function notation which makes it easier to identify the source of the funciton.